images/origin/system-container/config.json.template

```diff
@@ -70,15 +70,13 @@
 	    "source": "proc"
 	},
 	{
-	    "destination": "/dev",
-	    "type": "tmpfs",
-	    "source": "tmpfs",
-	    "options": [
-		"nosuid",
-		"strictatime",
-		"mode=755",
-		"size=65536k"
-	    ]
+		"destination": "/dev",
+		"type": "bind",
+		"source": "/dev",
+		"options": [
+			"rbind",
+			"rprivate"
```

Comment:
Thanks for investigating this.  It should not be a problem. Please change the noNewPrivileges to false if it addresses the SELinux issue